pages/guides/_guides.html

```diff
@@ -0,0 +1,54 @@
+<div class="searchbar">
+    <input autofocus class="search_input" type="text" placeholder="Search guides...">
+    <i class="fa fa-search search_icon"></i>
+</div>
+<div class="guide-filter">
+    <div class="col-lg-8">
+        <label>Filter Topic: </label>
+        <span class="badge badge-pill badge-default">AWS</span>
+        <span class="badge badge-pill badge-default">GCP</span>
+        <span class="badge badge-pill badge-default">GKE</span>
+    </div>
+    <div class="col-lg-4 text-right">
+         <label>Sort by</label>
+        <label class="dropdown">
+            <a href="#" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                Popular <span class="caret"></span>
+            </a>
+            <ul class="dropdown-menu">
+                <li><a href="#">GCP</a></li>
+                <li><a href="#">GKE</a></li>
+                <li><a href="#">AWS</a></li>
```

Comment:
Not sure it makes sense to sort by cloud. Only to filter by cloud, as per the three buttons to the left that you already have.

Reply:
Yeah true. A better way might be to sort alphabetically or chronologically